src/Microsoft.AspNetCore.Blazor/Components/BlazorComponent.cs

```diff
@@ -213,7 +213,7 @@ protected RenderTreeFrame bind(DateTime value, string format)
         /// <returns>A <see cref="RenderTreeFrame"/> that represents the event handler.</returns>
         protected RenderTreeFrame onclick(Action handler)
             // Note that the 'sequence' value is updated later when inserted into the tree
-            => RenderTreeFrame.Attribute(0, "onclick", _ => handler());
+            => RenderTreeFrame.Attribute(0, "onclick", handler != null ? (_ => handler()) : (UIEventHandler)null);
```

Comment:
I know this code will be redundant once #516 is merged, but it's needed until then for the E2E scenario to be valid